### ISSUE TITLE
Support for influxdb 2.0 metrics

### ISSUE THREAD

**Issue by destag** [go-kit/kit#1037]
Influx recently released [2.0](https://www.influxdata.com/blog/influxdb-2-0-open-source-is-generally-available/) version of their database. It brings some major changes (like [dropped support](https://docs.influxdata.com/influxdb/v2.0/upgrade/v1-to-v2/#supported-protocols) for udp protocol).

With new version they released a new golang [client](https://github.com/influxdata/influxdb-client-go). Can we add support for this new client?

I would be happy to implement it. But I wanted to find out if it is desired.

**Comment by peterbourgon** [comment]
I don't think code like this needs to live in the Go kit repo. Do you agree? If so, I would be happy to link to your implementation.

**Comment by destag** [comment]
There is metrics package for [influx 1.x](https://github.com/go-kit/kit/tree/master/metrics/influx) so I thought it would be nice if we had similar wrapper for influx 2.x

**Comment by sagikazarmark** [comment]
I agree with @peterbourgon: the core is large enough as it is. As part of #843 things like metrics should probably be moved out of the core repo.
